apps/api/src/app/integrations/dtos/credentials.dto.ts

```diff
@@ -115,4 +116,24 @@ export class CredentialsDto {
   @IsString()
   @IsOptional()
   webhookUrl?: string;
+
```

Comment:
missing attributes that i located due the `implements ICredentials` (in a hacky way)